docs/pages/integrations/github/apis/search_enterprise_audit_logs.mdx

```diff
@@ -0,0 +1,58 @@
+import { Callout } from "nextra/components";
+
+# Search Audit Logs for an Enterprise
+
+This method enables users to query the audit logs for a specified enterprise. Events include what action was performed,
+who performed the action, and when the action was performed.
+
+<Callout type="info">
+	For more information on the GitHub Audit logs API, see
+	[Here](https://docs.github.com/en/enterprise-cloud@latest/admin/monitoring-activity-in-your-enterprise/reviewing-audit-logs-for-your-enterprise/using-the-audit-log-api-for-your-enterprise).
+</Callout>
+
+<Callout type="info">
+	To use this API endpoint, you need to enable the **audit_log** scope in your
+	GitHub token.
+</Callout>
```

Comment:
I would prefer the following:

To use this API endpoint, your GitHub token requires the following scopes:

```
- audit_log
```